ambry-network/src/main/java/com/github/ambry/network/http2/Http2Utils.java

```diff
@@ -38,4 +41,24 @@ static RequestInfo releaseAndCloseStreamChannel(Channel streamChannel) {
     }
     return requestInfo;
   }
+
+  /**
+   * Get number of total pending tasks in Netty EventLoopGroup.
+   */
+  public static long getNumberOfPendingTasks(final EventLoopGroup EventLoopGroup) {
+    int index = 0;
+    long totalPendingTasks = 0;
+    for (final EventExecutor eventExecutor : EventLoopGroup) {
+      if (eventExecutor instanceof SingleThreadEventExecutor) {
+        int pendingTasks = ((SingleThreadEventExecutor) eventExecutor).pendingTasks();
+        totalPendingTasks += pendingTasks;
+        logger.debug("EventLoop-{} pending tasks: {}", index, pendingTasks);
+        index++;
+      } else {
+        logger.warn("EventLoop is not SingleThreadEventExecutor");
```

Comment:
It looks like the logger in this class is still set to `Http2ClientResponseHandler`